fluentd/generate_throttle_configs.rb

```diff
@@ -2,16 +2,35 @@
 require 'date'
 require 'logger'
 
+log_level = ENV['LOG_LEVEL'] || 'INFO'
```

Comment:
I think it should be 'WARN' given that is what we are defaulting to if its not a valid setting. It should be the same as when it isn't provided